test/test_street_address.rb

```diff
@@ -139,6 +140,17 @@ def test_parse
     assert_equal addr.state, 'WI'
     assert_equal addr.postal_code, '53949'
 
+    # not a perfect parse, but this asserts not getting wrong data or an error
+    addr = StreetAddress::US.parse(@addr9)
+    assert_equal addr.line1, ''
+    assert_equal addr.number, nil
+    assert_equal addr.street, nil
```

Comment:
I don't know if anyone wants to battle that regex, but I guess what would be nice here is if the street in the address string actually appeared here.

Reply:
I agree for sure-- that's why I felt compelled to leave the comment above the block here. I think this is enough of an edge case that I'm okay just getting the city/state/zip back, unless you feel like that violates the expectation from the return value of this gem.